runbot_travis2docker/models/runbot_repo.py

```diff
@@ -40,3 +40,4 @@ def weblate_validation(self):
             json = response.json()
             if 'projects' not in json:
                 raise ValidationError(_('Response json bad formated'))
+        raise Warning(_('Connection with weblate successful'))
```

Comment:
Maybe @JesusZapata could check how to work into ir_mail_server to do not rollback and if this Moy possible here the message in chatter i like it as well